pom.xml

```diff
@@ -6,7 +6,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>3.2.0</version>
+		<version>3.2.3</version>
```

Comment:
@beckermarc  I am not sure because of the GraalVM Native Image issues?

Reply:
3.2.2 had a issue that caused problems in Native Image for us. 3.2.3 has fixed this. The hibernate issue listed for 3.2.3 shouldn't be relevant for us.